MyLibrary/Sources/LiveTranslationFeature/LiveTranslationServiceClient.swift

```diff
@@ -0,0 +1,52 @@
+import Dependencies
+import DependenciesMacros
+import Foundation
+import LiveTranslationSDK_iOS
+
+extension DependencyValues {
+  public var liveTranslationServiceClient: LiveTranslationServiceClient {
+    get {
+      self[LiveTranslationServiceClient.self]
+    }
+    set {
+      self[LiveTranslationServiceClient.self] = newValue
+    }
+  }
+}
+
+@DependencyClient
+public struct LiveTranslationServiceClient {
+  public var langSet: @Sendable (String?) async throws -> LanguageEntity.Response.LangSet
+  public var langList: @Sendable () async throws -> [LanguageEntity.Response.LanguageItem]
+  public var chatRoomInfo: @Sendable (String) async throws -> ChatRoomEntity.Make.Response
+  public var chatConnection:
+    @Sendable (String) -> AsyncThrowingStream<RealTimeEntity.ChatStream, any Error> = { _ in .never
+    }
+  public var requestBatchTranslation:
+    @Sendable ([RealTimeEntity.Translation.Request.ContentData]) async -> Void
+}
+
+extension LiveTranslationServiceClient: DependencyKey {
+  static func live() -> Self {
```

Comment:
Use liveValue directly

Reply:
@d-date Rewrote it on [7d53241](https://github.com/tryswift/try-swift-tokyo/pull/97/commits/7d53241b9d8e05e821552e6b4e9d283403f92e25)